apps/judicial-system/backend/test/e2e.spec.ts

```diff
@@ -322,7 +323,7 @@ function expectCasesToMatch(caseOne: CCase, caseTwo: CCase) {
   expect(caseOne.otherRestrictions || null).toBe(
     caseTwo.otherRestrictions || null,
   )
-
+  expect(caseOne.isolationTo || null).toBe(caseTwo.isolationTo || null)
```

Comment:
isolationTo must also be handled in caseToCCase the same way other date fields are handled.